src/api/archive_unstable_storageDiff.md

```diff
@@ -0,0 +1,114 @@
+# archive_unstable_storageDiff
+
+**Parameters**:
+
+- `currentHash`: String containing a hexadecimal-encoded hash of the header of the block whose storage difference will be retrieved. The storage difference is calculated between the `currentHash` block and the parent of the `currentHash` block.
+
+- `fromHash` (optional): String containing a hexadecimal-encoded hash of the header of the block from which the storage difference will be calculated. When this parameter is provided, the storage difference is calculated between the `currentHash` block and the `fromHash` block. If this parameter is not provided, the storage difference is calculated between the `currentHash` block and the parent of the `currentHash` block.
+
+- `items` (optional): JSON object containing the following fields:
+
+    ```json
+    "items": {
+        "prefixes": [
+            {
+                "key": "0x...",
+                "type": "value" | "hash" | "none",
+            },
+        ],
+        "excludeKeyPrefixes": [
+            "0x...",
+        ],
+        "childTrie": "0x...",
+    }
+    ```
+
+  If the `items` parameter is not provided, the storage difference is calculated for all keys of the main storage trie.
+  - `prefixes` (optional): Array of JSON objects describing how the storage difference will be calculated. Each object contains the following fields:
+    - `key`: String containing the hexadecimal-encoded key prefix under which the storage difference is calculated.
+    - `type`: String equal to one of: `value`, `hash`, `none`.
+  - containing the key prefixes for which the storage difference will be calculated. If this parameter is not provided, the storage difference is calculated for all keys.
+  - `excludeKeyPrefixes` (optional): Array of strings containing the key prefixes for which the storage difference will not be calculated. If this parameter is not provided, the storage difference is calculated for all keys.
+  - `childTrie` (optional): A string containing the hexadecimal-encoded key of the child trie of the "default" namespace. If this parameter is not provided, the storage difference is calculated for the main storage.
```

Comment:
I guess that you just put this parameter here without giving much thoughts to it in order to get child tries out of the way, but it is very incoherent with the rest of the design of the function.
The function lets you query the difference between a range of blocks, but then you would have to query each child trie one by one? That's extremely weird to me.

Reply:
Yeap, I wondering as well if that's the right place to put it. Have extended it similar to:

```
items [
 {
    (optional) prefixes...
    trieType: mainTrie | childTrie
    (optional) childTrieKey
 } 
]
```

This should provide the storage difference of multiple tries. 
I've added the nonoptional `trieType` because it felt odd for users to say:
` items: [ { }, { "childKey": ...}]`  (ie have an empty object for main trie queries)